### PR TITLE
Content block simple content model

### PR DESCRIPTION
**Pull request by mfearer94**
I've added the content model and an anchor link to the content block simple component.

### Changelog

**New**

- Content model
- Anchor link

### CODE REVIEW

**Review by JennySanchez — APPROVED** [review]
Looks good to me. Thank you, Michael!

File: src/pages/components/content-block-simple.mdx [hunk 1]
```diff
@@ -32,9 +33,22 @@ Content block &mdash; simple with aside elements is also used for introducing pa
 
 ![Image of content block simple with aside elements](../../images/pattern/content-block-simple/content-block-simple-with-aside-elements.jpg)
 
+## Content guidance
+
+| Field | Field type | Required   | Cardinality | Max character size (English / translated) |
+| -------- | --------- | ---------- | ---------- | ----------|
+| Headline | Short copy | Yes | 1 | XXS (40 / 55)|
+| TOC headline override | Short copy | No | 1 | Mini (25 / 35)|
```
Comment (photodow) [inline src/pages/components/content-block-simple.mdx:41]: I saw this in another one. Is this a mistake copy over from something else?

File: src/pages/components/content-block-simple.mdx [hunk 2]
```diff
@@ -32,9 +33,22 @@ Content block &mdash; simple with aside elements is also used for introducing pa
...
+| Paragraph | Long copy | Yes | 1 | XXXXL (1600 / 2400)|
+| [Image with caption](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/image-with-caption) or [Video player](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/video-player) | Component | No | 1 | - |
+| [CTA (card)](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/cta) | Component | No | 1 | - |
+| [Link list](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/link-list) | Component | No | 1 | - |
```
Comment (photodow) [inline src/pages/components/content-block-simple.mdx:45]: Hmm... I might need to update functional specs to align with this. I still don't believe this should be apart of this component personally. I think it breaks the system we've created up to this point.

Comment (photodow) [inline src/pages/components/content-block-simple.mdx:45]: Heading is from content block, paragraph (it's also more then one paragraph btw) is from content item, and media is switched between video or image. Just calling that out here for what it's worth.

**Review by photodow — COMMENTED** [review]
Mostly curious about the TOC row.

File: src/pages/components/content-block-simple.mdx (hunk 2 above)
Comment (mfearer94) [reply src/pages/components/content-block-simple.mdx:45]: @photodow If we want to remove the link list integration, then I think we definitely need to get rid of the "Variation" section from this page.

Comment (mfearer94) [reply src/pages/components/content-block-simple.mdx:45]: Hmm yeah "paragraph" might not be the right term for this then. Maybe "description" or "long copy"? @photodow

File: src/pages/components/content-block-simple.mdx (hunk 1 above)
Comment (mfearer94) [reply src/pages/components/content-block-simple.mdx:41]: I just mentioned this in another PR

**Pull request merged**